src/app/Page.js

```diff
@@ -28,3 +62,54 @@ export default class Page extends React.Component<{|+children: Node|}> {
     );
   }
 }
+
+const style = StyleSheet.create({
+  header: {
+    color: "#1297A1",
+    fontFamily: '"Roboto Condensed", sans-serif',
+  },
+  body: {
+    padding: "0 100px",
+    flex: 3,
+  },
+  nav: {
+    height: 60,
+    padding: "20px 100px",
+  },
+  navItem: {
+    display: "inline-block",
+  },
+  navList: {
+    listStyle: "none",
+    paddingLeft: 0,
+    margin: 0,
+    display: "flex",
+  },
+  navLink: {
+    color: "#00848B",
+    fontFamily: "Roboto Condensed",
+    fontSize: 18,
+    textDecoration: "none",
+    ":hover": {
+      color: "#08A27F",
```

Comment:
fyi: This color `#08a27f` against white has a contrast ratio of just
3.24, which is technically too low for the font size at which it’s
rendered (minimum is 4.5). However, I’m okay with it given that it
appears only as a hover style, and that the main link style does have
sufficient contrast. Up to you whether to take any action.

Reply:
👍 I've swapped the colors again to ones with better contrast